src/components/TokenPairSelector/tokenPairSelector.styled.ts

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components'
```

Comment:
given the amount of markdown and CSS I wouldn't have split into two files, specially because I expect that we refactor this further creating a set of buttons and stories of buttons, so this Token Pair selector has minimal CSS/HTML